core/src/main/java/org/kiwix/kiwixmobile/core/help/HelpFragment.kt

```diff
@@ -17,83 +17,91 @@
-  protected open fun rawTitleDescriptionMap(): List<Pair<Int, Any>> = emptyList()
-  override val fragmentToolbar: Toolbar? by lazy {
-    fragmentHelpBinding?.root?.findViewById(R.id.toolbar)
-  }
-  override val fragmentTitle: String? by lazy { getString(R.string.menu_help) }
 
-  private val titleDescriptionMap by lazy {
-    rawTitleDescriptionMap().associate { (title, description) ->
-      val descriptionValue = when (description) {
-        is String -> description
-        is Int -> resources.getStringArray(description).joinToString(separator = "\n")
-        else -> {
-          throw IllegalArgumentException("Invalid description resource type for title: $title")
-        }
-      }
+  protected abstract val navHostFragmentId: Int
 
-      getString(title) to descriptionValue
+  // Instead of keeping the XML binding, we now directly return a ComposeView.
+  protected open fun createFragmentView(
+    inflater: LayoutInflater,
+    container: ViewGroup?
+  ): View {
+    return ComposeView(requireContext()).apply {
+      setContent {
+        // Create the helpScreen data using your rawTitleDescriptionMap.
+        val helpScreenData = transformToHelpScreenData(
+          requireContext(),
+          rawTitleDescriptionMap()
+        )
+        // Retrieve the NavController if your composable needs it.
+        val navController = Navigation.findNavController(requireActivity(), navHostFragmentId)
+        // Call your HelpScreen composable.
+        HelpScreen(data = helpScreenData, navController = navController)
+      }
     }
   }
 
+  // Each subclass is responsible for providing its own raw data.
+  protected open fun rawTitleDescriptionMap(): List<Pair<Int, Any>> = emptyList()
+
+  // The following properties are now optional – if no longer use an XML toolbar or title,
+  // we can remove or update these accordingly.
+  override val fragmentToolbar: Toolbar? by lazy {
+    // Already Applied ad TopAppBAr in scaffold in composable
+    null
+  }
+  override val fragmentTitle: String? by lazy { getString(R.string.menu_help) }
+
   override fun inject(baseActivity: BaseActivity) {
     (baseActivity as CoreMainActivity).cachedComponent.inject(this)
   }
 
+  // Remove or adjust onViewCreated if you no longer need to manipulate XML-based views.
   override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
     super.onViewCreated(view, savedInstanceState)
-    val activity = requireActivity() as AppCompatActivity
-    fragmentHelpBinding?.activityHelpDiagnosticImageView?.setOnClickListener {
-      sendDiagnosticReport()
-    }
-    fragmentHelpBinding?.activityHelpDiagnosticTextView?.setOnClickListener {
-      sendDiagnosticReport()
-    }
-    fragmentHelpBinding?.activityHelpRecyclerView?.addItemDecoration(
-      DividerItemDecoration(activity, DividerItemDecoration.VERTICAL)
-    )
-    fragmentHelpBinding?.activityHelpRecyclerView?.adapter = HelpAdapter(titleDescriptionMap)
```

Comment:
After moving `HelpScreen` to compose our `HelpAdapter` is unused so we should remove the unused code.